examples/bri-3/components/identity/bpiAccount/bpiAccount.ts

```diff
@@ -0,0 +1,12 @@
+import { IBpiAccount } from "./bpiAccount.interface";
+
+export class BpiAccount implements IBpiAccount {
+    id: string; // TODO: Add uuid after #491
+    nonce: number;
+    ownerBpiSubjectId: string;
+
+    constructor(nonce: number, ownerBpiSubjectId: string) {
+        this.nonce = nonce;
```

Comment:
we will probably add better way for nonce initialization, maybe some function that increments it, but for now maybe its enough to have it in constructor?

Reply:
It is enough to have it in constructor I believe

Reply:
@skosito ideally the constructor caller function must be having the nonce increment logic. So I think it is fine to have the constructor initializer